test/Microsoft.Dnx.Testing.SampleTests/DnuRestoreTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.IO;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.Dnx.Testing.SampleTests
+{
+    [Collection("SampleTestCollection")]
+    public class DnuRestoreTests : DnxSdkFunctionalTestBase
+    {
+        [Theory]
+        [MemberData(nameof(DnxSdks))]
+        public void DnuRestoreInstallsIndirectDependency(DnxSdk sdk)
+        {
+            // SimpleChain -> DependencyA -> DependencyB
+            const string appName = "SimpleChain";
+            const string solutionName = "DependencyGraphs";
+
+            var solution = TestUtils.GetSolution<DnuRestoreTests>(sdk, solutionName);
+            var project = solution.GetProject(appName);
+            var tempProjectDir = TestUtils.GetTempTestFolder<DnuRestoreTests>(sdk);
```

Comment:
The current implementation requires two copies of the solution. One is used to build the packages and created a feed. The other copy then uses this feed when restoring the SimpleChain project. It might be possible to simplify this though, I'll look into it.